data/filter/wg21.py

```diff
@@ -199,9 +205,17 @@ def tonytable(table, doc):
     def build_header(elem):
         # We use a `pf.RawInline` here because setting the `align`
         # attribute on `pf.Div` does not work for some reason.
-        header = pf.Plain(pf.RawInline('<div align="center">', 'html'),
+        header = pf.Null()
```

Comment:
Oh, I think I see what I was struggling with in the above comment now.
This can be simplified to this:
```
header = pf.Div(
    pf.Plain(pf.RawInline('\\begin{center}', 'latex'),
             pf.Strong(*elem.content),
             pf.RawInline('\\end{center}', 'latex')),
    attributes={'style': 'text-align:center'})
```